### PR TITLE
bugfix: indexer import types

### PR DESCRIPTION
**Pull request by tzaffi**
# Handle type update and pass indexer E2E tests

This PR results from fixing the type inconsistency for `EvalDelta` in `go-algorand-sdk` (https://github.com/algorand/go-algorand-sdk/pull/531) and passing the [failing nightly test in indexer](https://app.circleci.com/pipelines/github/algorand/indexer/3796/workflows/4016c18f-f298-443e-a887-8cc7dce246f7/jobs/5337).

## NOTE:
This should not be merged until https://github.com/algorand/go-algorand-sdk/pull/531 is merged and the `go.mod` in this project is updated.

### CODE REVIEW

File: go.mod
Comment (tzaffi) [inline go.mod:6]: TODO: Do not merge as is ^^^. Only merge after merging https://github.com/algorand/go-algorand-sdk/pull/531 and updating with `go get go-algorand-sdk`.

File: conduit/plugins/processors/filterprocessor/gen/generate.go
Comment (tzaffi) [inline conduit/plugins/processors/filterprocessor/gen/generate.go:62]: Cannot hand this array type [EvalDelta.SharedAccounts](https://github.com/algorand/go-algorand-sdk/blob/77dbcb81b8ce3ff34144e3a37cc110cc024e4642/types/block.go#L265)

**Review by Eric-Warehime — APPROVED** [review]

**Pull request merged**
